specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-03-01/managedClusters.json

```diff
@@ -5529,6 +5547,36 @@
       },
       "description": "Azure Defender settings for the security profile."
     },
+    "ManagedClusterStorageProfileDiskCSIDriver": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether to enable AzureDisk CSI Driver. The default value is true."
+        }
```

Comment:
Missing `version` field?

Reply:
`version` field is just in preview branch.